tests/golden/check-dump-config/.xrefcheck.yaml

```diff
@@ -0,0 +1,3 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
```

Comment:
I have found a missing newline at the end of this file:
```suggestion:-0+0
# SPDX-License-Identifier: MPL-2.0

```



<p align="right" data-meta="generated_by_instant-checks">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>